src/library/album/AlbumDetails.vue

```diff
@@ -9,9 +9,9 @@
       </h1>
       <p>
         by
-        <router-link :to="{name: 'artist', params: { id: album.artistId }}">
-          {{ album.artist }}
-        </router-link>
+        <Seperate>
+          <router-link v-for="artist in album.artists" :key="artist.id" :to="{name: 'artist', params: { id: artist.id }}">{{ artist.name }}</router-link>
```

Comment:
if i add linebreaks like the linter suggests, we get an extra space in the outout. how can i fix that?

![image](https://github.com/tamland/airsonic-refix/assets/6832539/63d2344c-cb58-466b-9401-acf936162212)
![image](https://github.com/tamland/airsonic-refix/assets/6832539/acb616fe-621f-4b74-bcec-2dadcf27db8d)

Reply:
I'm aware of this issue, you can ignore it. I think the vue compiler should remove it but doesn't for some reason..